### PR TITLE
fix create project

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs [hunk 1]
```diff
@@ -30,29 +31,34 @@ namespace LT.DigitalOffice.ProjectService.Business.Commands.Project
...
-    private readonly ICreateProjectValidator _validator;
+    private readonly ICreateProjectRequestValidator _validator;
     private readonly ILogger<CreateProjectCommand> _logger;
     private readonly IHttpContextAccessor _httpContextAccessor;
     private readonly IRequestClient<ICreateWorkspaceRequest> _rcCreateWorkspace;
     private readonly IRequestClient<ICreateWorkTimeRequest> _rcCreateWorkTime;
     private readonly IRequestClient<ICreateImagesRequest> _rcImages;
 
-    private void CreateWorkspace(string projectName, Guid creatorId, List<Guid> users, List<string> errors)
+    private async void CreateWorkspace(string projectName, List<Guid> usersIds, List<string> errors)
```
Comment (BelehovEgor) [inline src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs:43]: should we use use Task instead void?

**Review by BelehovEgor — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs [hunk 2]
```diff
@@ -30,29 +31,34 @@ namespace LT.DigitalOffice.ProjectService.Business.Commands.Project
...
-        if (!users.Contains(creatorId))
+        if (!usersIds.Contains(creatorId))
         {
-          users.Add(creatorId);
+          usersIds.Add(creatorId);
         }
 
-        var response = _rcCreateWorkspace.GetResponse<IOperationResult<bool>>(
-          ICreateWorkspaceRequest.CreateObj(projectName, creatorId, users), timeout: RequestTimeout.Default).Result;
+        var response = await _rcCreateWorkspace.GetResponse<IOperationResult<bool>>(
```
Comment (Glunky) [inline src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs:57]: var

File: src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs [hunk 3]
```diff
@@ -68,15 +74,15 @@ private void CreateWorkspace(string projectName, Guid creatorId, List<Guid> user
...
+    private async void CreateWorkTime(Guid projectId, List<Guid> userIds, List<string> errors)
     {
       string errorMessage = $"Failed to create a work time for project {projectId} with users: {string.Join(", ", userIds)}.";
       const string logMessage = "Failed to create a work time for project {projectId} with users {userIds}";
 
       try
       {
-        var response = _rcCreateWorkTime.GetResponse<IOperationResult<bool>>(
-          ICreateWorkTimeRequest.CreateObj(projectId, userIds)).Result;
+        var response = await _rcCreateWorkTime.GetResponse<IOperationResult<bool>>(
```
Comment (Glunky) [inline src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs:84]: var

File: src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs [hunk 4]
```diff
@@ -103,19 +109,24 @@ private List<Guid> CreateImage(List<ImageContent> projectImages, Guid userId, Li
 
       try
       {
-        IOperationResult<ICreateImagesResponse> response = _rcImages.GetResponse<IOperationResult<ICreateImagesResponse>>(
+        var response = await _rcImages.GetResponse<IOperationResult<ICreateImagesResponse>>(
```
Comment (Glunky) [inline src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs:112]: var

**Review by VLiamin — APPROVED** [review]

**Pull request merged**